about.html

```diff
@@ -1,111 +1,100 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="desktop.css">
-    <link rel="stylesheet" href="about.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
-<link rel="icon" href="Resources/images/fastdevz-icon.png">
-    <title>FASTDEVZ INC</title>
-</head>
-<body>
-     <!-- header with hav in it & the header need to be in the javascript object -->
-     <header id="nav">
-        <!-- use only in desktop-->
-            <div class="header-bar">  
-            </div>
-        <!-- end-->
-        <nav>
-            <a class="logo" href="index.html"><img src="Resources/images/my-logo.png" alt="fastdevz logo"></a>
-            <img src="Resources/images/menu-icon-1.png" alt="tagle-menu">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="about.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>About Fastdevz</title>
+  </head>
+  <body>
+    <header>
+      <div class="topMenu">
+        <nav id="topNav">
+          <ul>
+            <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href=""><i class="fab fa-twitter"></i></a></li>
+            <li><a href="">English</a></li>
+            <li><a href="">My Page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
         </nav>
+      </div>
+      <div class="mainMenu">
+        <div class="mainMenuCont">
+          <div class="logo">
+            <a href="index.html"><img src="Resources/images/my-logo.png" alt=""></a>
+          </div>
+          <i id="menuBtn" class="fas fa-bars fa-lg"></i>
+          <nav id="mainNav">
+            <i id="closeBtn" class="fas fa-times fa-lg"></i>
+            <ul>
+              <li><a class="aboutLink" href="about.html">About</a></li>
```

Comment:
- [ ] It'll be nice if, on the about page, instead of the `about` nav link, you have a `home` nav link.

Reply:
done

Reply:
@phelian23 all done here